pyproject.toml

```diff
@@ -1,6 +1,4 @@
 [tool.black]
-# WARNING: This should be the same as in `.pre-commit-config.yaml`.
-required-version = "21.9b0"
```

Comment:
Can we change this now to `required-version = "22"`

Reply:
Just re-added. Do we need to re-add stuff to the pre-commit hook too, or is that covered by this?